get_member_userid.py

```diff
@@ -45,7 +45,7 @@
 
 chat = build("chat", "v1", http=http_auth)
 
-response = chat.spaces().members().list(parent=THOTH_DEVOPS_SPACE)
+response = chat.spaces().members().list(parent="spaces/AAAAVjnVXFk")
```

Comment:
hmmm... what about setting this in the var definition instead? Thus leaving the option to override via env var:

https://github.com/AICoE/sefkhet-abwy/blob/653be9740a2650d22c34e41c85ab56944a66ed48/get_member_userid.py#L38

i.e. turn this into:
``` python
 THOTH_DEVOPS_SPACE = os.getenv("SESHETA_THOTH_DEVOPS_SPACE", "spaces/AAAAVjnVXFk") 
```

Reply:
oh ja, I was lazy :) solved with 1969ac7